### ISSUE TITLE
Error initializing plugin

### ISSUE THREAD

**Issue by thoward27** [Fullstak-nl/medusa-plugin-postmark#6]
I am seeing the following error when trying to enable this plugin

```
warn:    An error occurred while registering API Routes in medusa-plugin-postmark
Error: Cannot find module '../../generators/pdfGenerator'
```

Looking at the module in my source tree, I suspect it's `api/routes/store/export/route.js`, as that uses the same `require` even though it is a layer deeper than the other references to the generator. 

```
api/store/export/route.js
10:var _pdfGenerator = _interopRequireDefault(require("../../generators/pdfGenerator"));

api/routes/export/route.js
10:var _pdfGenerator = _interopRequireDefault(require("../../generators/pdfGenerator"));

api/routes/store/export/route.js
10:var _pdfGenerator = _interopRequireDefault(require("../../generators/pdfGenerator"));
```

Best,
Tom

**Comment by breakerh** [comment]
Hello @thoward27 seems like my build didn't clear some files sorry.
Will push the fix later tonight.

Thanks for letting me know.

**Comment by Fullstak-nl** [comment]
Fixed in V4.6.1
https://github.com/Fullstak-nl/medusa-plugin-postmark/releases/tag/v4.6.1
